nucliadb_telemetry/nucliadb_telemetry/logs.py

```diff
@@ -176,40 +224,47 @@ def setup_logging() -> None:
         if logger_name not in settings.logger_levels:
             settings.logger_levels[logger_name] = level
 
-    formatter = JSONFormatter()
-    json_handler = logging.StreamHandler()
-    json_handler.setFormatter(formatter)
-    stream_handler = logging.StreamHandler()
-    stream_handler.setFormatter(
-        logging.Formatter(
-            "[%(asctime)s.%(msecs)02d] [%(levelname)s] - %(name)s - %(message)s"
+    log_handler: logging.Handler
+    error_log_handler: logging.Handler
+    if settings.log_output_type == LogOutputType.FILE:
+        _maybe_create_file_directory(settings.info_log)
+        _maybe_create_file_directory(settings.error_log)
+        log_handler = RotatingFileHandler(
+            settings.info_log, maxBytes=settings.max_log_file_size, backupCount=5
         )
-    )
+        error_log_handler = RotatingFileHandler(
+            settings.error_log, maxBytes=settings.max_log_file_size, backupCount=5
+        )
+    else:
+        log_handler = error_log_handler = logging.StreamHandler()
```

Comment:
careful! I think `log_handler` and `error_log_handler` will be the same instance here.
```python
>>> class A:
...     ...
... 
>>> a = b = A()
>>> a is b
True
```

Reply:
yup, that's on purpose.

We only end up using the error log handler when we're outputting to an error log file